.circleci/config.yml.jinja

```diff
@@ -17,7 +25,15 @@ jobs:
             xcode: "10.2.0"
         steps:
             - checkout
-            - run: git submodule sync && git submodule update --init --recursive
+            - run:
+                  name: Update OPAM repo
+                  command: opam update
+            - run:
+                  name: Update OPAM packages
+                  command: opam switch import src/opam.export
+            - run:
+                  name: Update and pin external submodules
+                  command: ./scripts/pin-external-submodules.sh
```

Comment:
this job doesn't build any protocol code and doesn't use `opam`; can you remove the jobs here?

Reply:
already done